frontend/app/routes/_public+/apply+/$id+/terms-and-conditions.tsx

```diff
@@ -0,0 +1,173 @@
+import { ActionFunctionArgs, LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { CollapsibleDetails } from '~/components/collapsible';
+import { InlineLink } from '~/components/inline-link';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('terms-and-conditions');
+
+export const handle = {
+  breadcrumbs: [
+    {
+      labelI18nKey: 'terms-and-conditions:terms-and-conditions.index.breadcrumbs.terms-and-conditions',
+    },
+  ],
+  i18nNamespaces,
+  pageTitleI18nKey: 'terms-and-conditions:terms-and-conditions.index.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id, state } = await applyFlow.loadState({ request, params });
+  return json({ id, state: state.termsAndConditions });
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id } = await applyFlow.loadState({ request, params });
+
+  const sessionResponseInit = await applyFlow.saveState({
+    request,
+    params,
+    state: {},
+  });
+
+  return redirect(`/apply/${id}/personal-info`, sessionResponseInit);
+}
+
+export default function TermsAndConditions() {
+  const { id } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+  return (
+    <>
+      <p className="text-xl">{t('terms-and-conditions:terms-and-conditions.index.intro-text')}</p>
```

Comment:
Our content is sized at `16px` (equivalent to `1rem` of the user's browser font-size). Add margin-bottom classes such as `mb-6` or `mb-8` applied for spacing. Avoid the use of `<br />` for spacing.

```suggestion
      <p className="mb-6">{t('terms-and-conditions:terms-and-conditions.index.intro-text')}</p>
```

Reply:
Fixed: ed62887534ecb1a1d9810f348c88031cfc813d39